<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>331</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Adicionar</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_Refrigerado">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Refrigerado</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_Quentes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Quente</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_Snack">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Snack</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>156</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Quantidade:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>196</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nome">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>9</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Escolha o tipo do alimento:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lb_temp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>236</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Temperatura:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_temp">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lb_val">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>236</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Validade:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_validade">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lb_sabor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>236</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sabor:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_sabor">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="le_quantidade">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999999</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_show">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar todos itens</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_media">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>MÃ©dia da quantidade</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>351</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListView" name="listView_menu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>331</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Estoque&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
